dstep/driver/Application.d

```diff
@@ -37,20 +38,24 @@ class Application : DStack.Application
 
     private
     {
-        Language language;
+        Options blueprint;
```

Comment:
`blueprint` holds default values, then I need to change some values before passing it to subsequent processing. I thought blueprint in the meaning of draft is a good name.